cli/wallet/validator.go

```diff
@@ -0,0 +1,100 @@
+package wallet
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/cli/flags"
+	"github.com/nspcc-dev/neo-go/cli/options"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/client"
+	"github.com/nspcc-dev/neo-go/pkg/vm/emit"
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+	"github.com/urfave/cli"
+)
+
+func newValidatorCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:      "vote",
+			Usage:     "vote for a validator",
+			UsageText: "vote -w <path> -r <rpc> [-t <timeout>] [-g gas] -a <addr> -c <public key>",
```

Comment:
Isn't it `[-s <timeout>]` (not `-t`)? [link](https://github.com/nspcc-dev/neo-go/blob/master/cli/options/options.go#L34)

Reply:
Fixed.